fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -1291,6 +1295,7 @@ void ProcessClassRangeImpl::clear(ReadYourWritesTransaction* ryw, const KeyRef&
 }
 
 ACTOR Future<RangeResult> getProcessClassSourceActor(ReadYourWritesTransaction* ryw, KeyRef prefix, KeyRangeRef kr) {
+	ryw->setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
```

Comment:
As the `transaction` is not requiring options to read system keys,
this is for explicitly setting the option the RYW?

Reply:
The transaction is reading system keys, and this fact is now important at the native level because reading the system keys implies raw access. I suppose maybe this is an awkward situation for the special keys API, since we are enabling system key access for external transactions. I guess we could change the logic here and in the rest of this class to use `RAW_ACCESS` instead.

Reply:
I'll go ahead and merge this because the problem is a bit more widespread, but I can do the update in a different PR.